provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
```

Comment:
Hi Paolo, I am trying this comment functionality. 
Actually there is a reason, because declaration of a provider we need only once. This file is not needed, provider is in main already.